runtime/lua/vim/filetype/detect.lua

```diff
@@ -43,10 +43,27 @@ function M.btm(bufnr)
   end
 end
 
-function M.cfg(path, bufnr)
+-- Returns true if file content looks like RAPID
+local function is_rapid(bufnr, extension)
+  if extension == "cfg" then
+    local line = getlines(bufnr, 1):lower()
+    return findany(line, { "eio:cfg", "mmc:cfg", "moc:cfg", "proc:cfg", "sio:cfg", "sys:cfg" })
+  end
+  local first = "^%s*module%s+%S+%s*"
```

Comment:
Is ignoring the keyword argument ok in favor of `%S`?

Reply:
This is tricky. `\k` might be too difficult to translate into pure Lua, so this may be a case where `vim.regex` is warranted.